api/infrastructure/parallelcluster-api.yaml

```diff
@@ -166,7 +166,7 @@ Resources:
         - UseCustomPoliciesTemplateUri
         - !Ref PoliciesTemplateUri
         - !Sub
-          - "s3://${AWS::Region}-aws-parallelcluster/parallelcluster/${Version}/templates/policies/policies.yaml"
+          - "https://${AWS::Region}-aws-parallelcluster.s3.amazonaws.com/parallelcluster/${Version}/templates/policies/policies.yaml"
```

Comment:
This doesn't work in partitions other than commercial, check `get_templates_bucket_path`